test/e2e/user_workload_monitoring_test.go

```diff
@@ -97,34 +105,7 @@ func TestUserWorkloadMonitoringMetrics(t *testing.T) {
 			name: "assert UWM federate endpoint is exposed",
 			f:    assertUWMFederateEndpoint,
 		},
-	} {
-		t.Run(scenario.name, scenario.f)
-	}
-}
-
-func TestUserWorkloadMonitoringAlerting(t *testing.T) {
```

Comment:
merged them in `TestUserWorkloadMonitoring` to make use of the same setup and be able to run them in //